README.md

```diff
@@ -22,6 +22,12 @@ https://github.com/tanishiking/jpp/releases
 $ go get -u github.com/tanishiking/jpp/cmd/jpp
 ```
 
+### Develop
+```
+$ curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+$ dep ensure
```

Comment:
Ah, sorry I just merged [this PR](https://github.com/tanishiking/jpp/pull/7) that let us use `go mod` instead of `dep`.
And, basically, I recommend developers to use `make` commands instead :)